tests/layer_tests/tensorflow_tests/test_tf_FloorDiv.py

```diff
@@ -49,3 +48,65 @@ def test_add_placeholder_const_1D(self, params, ie_device, precision, ir_version
                                                           use_new_frontend=use_new_frontend),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_new_frontend=use_new_frontend)
+
+
+class TestFloorDivStaticInput(CommonTFLayerTest):
+    def create_flordiv_tf_net(self, min, max, step, y, dtype, ir_version, use_new_frontend, **kwargs):
+        import tensorflow as tf
+        
+        x = np.arange(min, max, step, dtype=dtype)
+        x_shape = x.reshape(kwargs['x_shape']).shape if 'x_shape' in kwargs else x.shape
+
+        tf.compat.v1.reset_default_graph()
+
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(dtype, x_shape, 'Input')
+            y = tf.constant(np.array(y).astype(dtype))
+            res = tf.raw_ops.FloorDiv(x=x, y=y)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+    
+    def _prepare_input(self, inputs_dict, kwargs):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.arange(kwargs['min'], kwargs['max'], kwargs['step'], dtype=kwargs['dtype'])
```

Comment:
please save `min` and other values in self corresponding the object. Avoid modifying `kwargs`

Reply:
but i didn't modify `kwargs` at all. i just pass them with the currently existing mechanism with `kwargs_to_prepare_input`: https://github.com/openvinotoolkit/openvino/blob/7423fc52a42a441509727b06c0907143a3481a51/tests/layer_tests/common/layer_test_class.py#L88-L92

It's already used e.g.
https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/tensorflow_tests/test_tf_FakeQuantize.py#L142-L148
https://github.com/openvinotoolkit/openvino/blob/7423fc52a42a441509727b06c0907143a3481a51/tests/layer_tests/pytorch_tests/test_arange.py#L121-L123

Reply:
i would've been ideal to do like this without mutable dicts, 
![image](https://github.com/openvinotoolkit/openvino/assets/5703039/5e5074c9-c73d-436f-aa2f-7e658ed68c84)

but unfortunately in order to do so, we should modify common tests here  `kwargs['kwargs_to_prepare_input']` -> `**kwargs['kwargs_to_prepare_input']` https://github.com/openvinotoolkit/openvino/blob/7423fc52a42a441509727b06c0907143a3481a51/tests/layer_tests/common/layer_test_class.py#L88-L90

like it's for torch layer tests e.g. in torch 
https://github.com/openvinotoolkit/openvino/blob/7423fc52a42a441509727b06c0907143a3481a51/tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py#L51

Can we consider this improvements separately and merge this PR as is?

Reply:
We have one day before code freeze and this change should take less one hour. So please fix it in this PR.